README.md

```diff
@@ -165,6 +165,27 @@ MarkLogic Server Docker containers are configured using a set of environment var
 | LICENSE_KEY           | license key                     | no                                | n/a       | set MarkLogic license key                          |
 | LICENSEE            | licensee information            | no                                | n/a       | set MarkLogic licensee information                 |
 |INSTALL_CONVERTERS   | true                            | no                                | false     | Installs converters for the client if they are not already installed | 
+|OVERWRITE_ML_CONF   | true                            | no                                | false     | Deletes and rewrites `/etc/marklogic.conf` with the passed in env variables if set to true | 
+
+
+MarkLogic Server also can be configured through a configuration file on the image to `/etc/marklogic.conf`. To interface with the configuration file pass in `OVERWRITE_ML_CONF` set to `true` and the following env variables can be written, if set, to `/etc/marklogic.conf`. 
+
+
+ | env var                       | value                           | required                          | default   | description                                        |
+| ------------------------------- | --------------------------------- | ----------------------------------- | ----------- | ---------------------------------------------------- |
+| MARKLOGIC_USER                | daniel                            | no                                |     n/a      | The username running MarkLogic within the docker container           |
```

Comment:
I would move MARKLOGIC_USER, MARKLOGIC_DISABLE_JVM, JAVA_HOME, CLASSPATH to the bottom of the list and move ML_HUGEPAGES_TOTAL above them. Those settings are more "advanced" and would only be used if building a new image with this as the base.

Reply:
See: https://github.com/marklogic/marklogic-docker/commit/be2b425cb7ccc9d7a4fde23dd06317b534b48c82 for the new format. LMK what you think, (copy from the previous comment on the env variables)